recipes-devtools/firmware-package-tool/firmware-package-tool.bb

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+DESCRIPTION = "Generate The Firmware Update Package"
+MAINTAINER = "huaqian.li@siemens.com"
+
+SRC_URI = "file://update.conf.json \
+           file://iot2050-fwu-update-json.sh"
+
+inherit dpkg-raw
+
+do_deploy_deb[nostamp] = "1"
+
+do_deploy_deb() {
+    # Generate the firmware update package
+    sh ${WORKDIR}/iot2050-fwu-update-json.sh ${WORKDIR} ${DEPLOY_DIR_IMAGE} $(${ISAR_RELEASE_CMD})
+}
+do_deploy_deb[dirs] = "${DEPLOY_DIR_IMAGE}"
```

Comment:
This is missing dependencies: If you only build this package, the firmware artifacts won't be there.

Reply:
Another reason to fix the u-boot build first - good to know. In any case, this workaround is not nice at all.